index.html

```diff
@@ -59,6 +60,214 @@ <h3 class="connect">LET'S CONNECT</h3>
+            </button>
+          </div>
+
+          <!--Card Three-->
+          <div class="first-card">
+            <div>
+              <img src="./images/Snapshoot Portfolio (1).png" alt="">
+            </div>
+
+            <div>
+              <h1>Tonic</h1>
+            </div>
+
+            <div class="details">
+              <h3>CANOPY</h3>
+              <img src="./images/Counter.png" alt="">
+              <h4>Back End Dev</h4>
+              <img src="./images/Counter.png" alt="">
+              <h4>2015</h4>
+            </div>
+
+            <p>
+              A daily selection of privately personalised reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+
+            <button class="see-project"><a href="#">
+              See Project</a>
+            </button>
+          </div>
+
+          <!--Card Four-->
+          <div class="first-card">
+            <div>
+              <img src="./images/Snapshoot Portfolio (3).png" alt="">
+            </div>
+
+            <div>
+              <h1>Multi-Post<br>Stories</h1>
+            </div>
+
+            <div class="details">
+              <h3>CANOPY</h3>
+              <img src="./images/Counter.png" alt="">
+              <h4>Back End Dev</h4>
+              <img src="./images/Counter.png" alt="">
+              <h4>2015</h4>
+            </div>
+
+            <p>
+              A daily selection of privately personalised reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+
+            <button class="see-project"><a href="#">
+              See Project</a>
+            </button>
+          </div>
+        </section>
+
+        <!--About Myself-->
+        <section>
+          <header class="me">
+            <h2>About<br>Myself</h2>
+          </header>
+
+          <p>Hello i'm a software developer! I can help<br>you build a product, feature or website<br>Look through some of my work and<br>experience! If you like what you see and have a project you need coded, don't<br>hesitate to contact me.</p>
+        
+          <div>
+            <h3 class="connect">LET'S CONNECT</h3>
+          </div>
+
+          <ul class="icon-logos">
+            <li class="icons"><a href="#"><i class="bi bi-twitter"></i></a></li>
+            
+            <li class="icons"><a href="#"><i class="bi bi-linkedin"></i></a></li>
+            
+            <li class="icons">
+              <img class="ang-logo" src="./images/angellist.svg" alt="">
+            </li>
+            
+            <li class="icons"><a href="#"><i class="bi bi-github"></i></a></li>
+            
+            <li class="icons">
+              <img class="ang-logo" src="./images/medium.svg" alt="">
+            </li>
```

Comment:
- [ ] Kindly ensure that the `alt` attribute has the meaningfull description. Alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic they cannot see.